analytics/views.py

```diff
@@ -1092,12 +1116,25 @@ def support(request: HttpRequest) -> HttpResponse:
 
         context["realms"] = realms
 
+        cnfs = []  # type: List[Dict[str, Any]]
+        preregistration_users = PreregistrationUser.objects.filter(email__in=key_words, status=0,
```

Comment:
let's remove `status` from the filter.
What is the `referred_by` for?

Reply:
Removed status. I am using it for diffrentiating between invites and normal sign up preregistration objects. refereed_by would not be None for invites. Actually I think I can combine both the queries. Let me do that.

Reply:
Should I should "link used" or something similiar for objects whose status is set to 1?

Reply:
yeah, that sounds great. (And "link unused" in the other case.)